xm-commons-i18n/src/main/java/com/icthh/xm/commons/i18n/error/domain/vm/FieldErrorVM.java

```diff
@@ -27,7 +27,14 @@ public void add(String objectName, String field, String message) {
         if (fieldErrors == null) {
             fieldErrors = new ArrayList<>();
         }
-        fieldErrors.add(new FieldErrorVM.FieldError(objectName, field, message));
+        fieldErrors.add(new FieldErrorVM.FieldError(objectName, field, message, null));
```

Comment:
I think it is better to use overloaded method `add()` instead of copying the logic